joinmarket/configure.py

```diff
@@ -116,11 +116,16 @@ def jm_single():
 socks5_host = localhost
 socks5_port = 9050
 #for tor
-#host = 6dvj6v5imhny3anf.onion
```

Comment:
Using hidserv publicly reveals the fact you're using tor, plus makes your bot unstable. Use HS only on irc networks which don't allow direct tor connections. Opionated I know, I'll revert this if you strongly disagree.